### ISSUE TITLE
Crashes, build problem and contribution

### ISSUE THREAD

**Issue by athach** [tlongstretch/freeserf-with-AI-plus#77]
Hi,
It is great to see someone keeping this project alive after jonls stop updating. I keep returning to this superb game once in a while but each time I get frustrated and annoyed because it always crashes and all progress is lost and I tell myself I should never give it a try again. But here I am again.

Unfortunately also your latest build crashes without any error message. First time was trying to build a road. Second time I believe I was just scrolling.

I am an experienced programmer and I was thinking I may be able to contribute. Unfortunately the hacking file is not very helpful for a Windows build environment. Fortunately I found the Wiki page https://github.com/freeserf/freeserf/wiki/Building-from-Source-on-Windows. With that I was able to create a Visual Studio solution. In that I get three errors: "cannot open file 'googletest_bin\Debug\lib\gtest.lib'". Any ides on how to fix that?

**Comment by p1plp1** [comment]
omg, so dont build googletest + build everything around, i just leaved M$ visual studio for bad expirience, im building with mingw ...

**Comment by tlongstretch** [comment]
yes, I just commented out googletest from my CMakeFiles.  @athach , I have moved this project to here:  https://github.com/forkserf

try out the stable branch if should have many of these bugs fixed
